settings.gradle.kts

```
@@ -2,6 +2,7 @@ rootProject.name = "telegram-bot"
 
 include(
     "internal-api",
-    "telegram-bot-api",
-    "finance-service-api"
+    "telegram-bot",
+    "finance-service",
+    "finance-models-api"
```

```suggestion
    "finance-models"
```
?

Mongo models are not the part of API

---

I realized the mistake, I'll look into the API topic more